src/breadcrumbs/_base.scss

```diff
@@ -24,13 +24,18 @@
 
 .breadcrumbs-list-item {
   display: inline-block;
-  margin: 0 ms(-7);
+  margin-right: ms(-7);
 
-  &:first-child {
-    margin-left: 0;
+  &:after {
+    content: '>';
```

Comment:
Don't forget that we want to use unicode here to ensure we're okay in other languages!